src/Http/OpenApi/src/Microsoft.AspNetCore.OpenApi.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>aspnetcore;openapi</PackageTags>
+    <Description>Provides APIs for annotating route handler endpoints in ASP.NET Core with OpenAPI annotations.</Description>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.OpenApi" />
+    <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.Routing" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.Core" />
```

Comment:
I posted a response on the other PR that I closed.

The requirement for a dependency on MVC largely stems from some carry-over in the way we generated Response annotations in this library. Moving forward, we want to simplify this logic (see https://github.com/dotnet/aspnetcore/issues/41243) and remove the dependency on the types defined in MVC.

As I typed this out, I realized that we also have a long-term plan to include methods for converting `OpenApiOperation` back to MVC types for backwards compat, although I'm not enthused by those plans anymore.

Reply:
@davidfowl mentioned that it might considering moving this to its own top-level directory under the `src` repo. That feels a little too heavy duty for me but might make the most sense considering this is not part of the framework anyways (e.g. avoid assuming it is a primitive like Http is).

Reply:
Moving to a top-level directory might also help to remove the MVC projects from the `HttpAbstractions.slnf`, right?

Reply:
OK. I've moved it to a top-level directory.